static/app/components/updatedEmptyState.tsx

```diff
@@ -43,32 +28,26 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
     p => p.id === currentPlatformKey
   ) as PlatformIntegration;
 
-  const platformPath = getPlatformPath(currentPlatform);
-
-  const module = useLoadGettingStarted({
-    platformId: currentPlatform.id,
-    platformPath,
+  const loadGettingStarted = useLoadGettingStarted({
+    platform: currentPlatform,
+    orgSlug: organization.slug,
+    projSlug: project?.slug,
   });
 
   if (
     !project ||
-    projectKeysIsError ||
-    projectKeysIsLoading ||
-    !projectKeys ||
-    projectKeys.length === 0 ||
-    !module ||
-    !currentPlatform ||
-    module === 'none'
+    loadGettingStarted.isError ||
+    loadGettingStarted.isLoading ||
+    !loadGettingStarted.docs ||
+    !loadGettingStarted.cdn ||
+    !loadGettingStarted.dsn
```

Comment:
do we still need to check for `!currentPlatform` here? it was checked in the old code

Reply:
I have added it back, but if currentPlatform is not available, then there won't be any docs, DSN, etc., available either. So, it should work just fine